### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_WCAG2.1_Homepage_Screenreader: When we launch narrator and navigate to “Search box” present at the top of the page and enter any valid input then narrator doesn’t announce the status message that is how many search items found.

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-docs#3434]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shbagl%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847575447&sdata=XQqvaO2rFh7p/7F2DmzFj2wgJSiYHGuJV73Xg8hPVbA%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Benchmark; #CELA P1; #ChromiumEdge; #Narrator; #DesktopWeb; #MAS4.1.3; #Win10; #Visual Studio Code; #CELA P1_A11y_code.visualstudio.com_WCAG2.1: #NVDA

**Environment Details:**
Application Name:  code.visualstudio.com
#URL:https://code.visualstudio.com/docs/editor/versioncontrol
Windows Version: Windows10
Browser version: Microsoft Edge 80.0.361.5
Chrome:Version 80.0.3987.106

**Repro Steps:** 
1.	Launch the application URL: https://code.visualstudio.com/docs/editor/versioncontrol
2.	Home screen will open
3.	Launch screen reader as narrator and navigate to “Search box” present at the top of the page.

**Actual:**
When we launch narrator and navigate to “Search box” present at the top of the page and enter any valid input then narrator doesn’t announce the status message that is how many search items found.

**The same issue is observed with NVDA**

**Expected:**
When we launch narrator and navigate to “Search box” present at the top of the page and enter any valid input then narrator should announce the status message that is how many search items found.

**User Impact:**
Screen reader users will not know of the change that took place which did not receive focus.

**Attachment:**
[#3434.zip](https://github.com/microsoft/vscode-docs/files/4265454/3434.zip)

**Comment by octref** [comment]
> When we launch narrator and navigate to “Search box” present at the top of the page and enter any valid input then narrator should announce the status message that is how many search items found.

Users can only find "how many search items found" after pressing enter to search. We do not do search-as-you-type so this isn't an accessibility issue.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.75
![image](https://user-images.githubusercontent.com/85863999/217254776-3b2b044c-2a0a-4862-b942-bd10bde4b94f.png)

**Comment by 12-shweta** [comment]
#CELA_code.visualstudio.com_WCAG2.1_FEB2020;
